definition/data/sheets/AuthorisationCaseField.json

```diff
@@ -252,6 +261,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "channel", "UserRole": "citizen", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "channel", "UserRole": "letter-holder", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "channel", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "responseDeadlinePreview", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
```

Comment:
this should be CRUD?

Reply:
this field as of permission is only available for caseworker, so no harm to have full access, however I don't know about this story

Reply:
Addressed - made the field read-only.